STM32H7B0VBTx_FLASH.ld

```diff
@@ -156,12 +156,13 @@ SECTIONS
   .data : 
   {
     . = ALIGN(4);
+    /* Note: _sdata and _edata are placed together to disable data initialization from FLASH */
     _sdata = .;        /* create a global symbol at data start */
+    _edata = .;        /* define a global symbol at data end */

```

Comment:
Right, this turns the memcpy in the startup script into basically a NOP, right?

Reply:
Yes, it does exactly that. I wasn't sure whether to edit startup_stm32h7b0xx.s as it wasn't tracked (I just provided a symlink to STM32CubeH7) so I assumed that an stock copy was used.

Reply:
No please keep this, it's great. This way we won't have to change the startup script and so on.

Reply:
Cool. It'll stay as is then :-)